test/users/npadmana/blas/test_blas3.chpl

```diff
@@ -0,0 +1,519 @@
+use Random;
```

Comment:
Unfortunately, no. I suspect porting those tests over would be significantly more work.... The current tests are really designed to just test out the Chapel interfaces --- I don't explicitly exercise the C-API, other than the fact that the Chapel code calls into it. And even with the Chapel interfaces, I've restricted myself to sanity checks, as opposed to complete code coverage.

I've taken the position that someone else has validated the BLAS code, and that our job here is really just to make sure that the Chapel interfaces are all fine. 

A code path that isn't currently exercised is the column-major storage option. There are some sanity tests that can be done (with square matrices here), but I've otherwise just assumed that everyone will use the Chapel row-major default.